plugin/trino-memory/src/main/java/io/trino/plugin/memory/MemoryMetadata.java

```diff
@@ -152,17 +152,10 @@ public synchronized void dropSchema(ConnectorSession session, String schemaName,
     @GuardedBy("this")
     private boolean isSchemaEmpty(String schemaName)
     {
-        if (tables.values().stream()
-                .anyMatch(table -> table.getSchemaName().equals(schemaName))) {
-            return false;
-        }
-
-        if (views.keySet().stream()
-                .anyMatch(view -> view.getSchemaName().equals(schemaName))) {
-            return false;
-        }
-
-        return true;
+        return tables.values().stream()
```

Comment:
nit:
```
        return Streams.concat(
                tables.values().stream().map(TableInfo::getSchemaName), 
                views.keySet().stream().map(SchemaTableName::getSchemaName))
                .anyMatch(schemaName::equals);
```

formats nicer

Reply:
actually it should be negated

Reply:
i will do
```
return tables.values().stream().anyMatch(table -> table.getSchemaName().equals(schemaName)) ||
                views.keySet().stream().anyMatch(view -> view.getSchemaName().equals(schemaName));
```

i think it's the most readable version